examples/mfem/bp1.hpp

```diff
@@ -255,5 +255,11 @@ class CeedMassOperator : public mfem::Operator {
     CeedVectorSetArray(u, CEED_MEM_HOST, CEED_USE_POINTER, x.GetData());
     CeedVectorSetArray(v, CEED_MEM_HOST, CEED_USE_POINTER, y.GetData());
     CeedOperatorApply(oper, u, v, CEED_REQUEST_IMMEDIATE);
+
+    const CeedScalar* array;
```

Comment:
Did you want to similarly comment this as to be updated with CeedVectorSyncArray?

Reply:
TODO comment added.